### ISSUE TITLE
TTA

### ISSUE THREAD

**Issue by a13383622838** [HeinzJiao/RoIPoly#4]
Thank you for your codes. I would like to know if ROIPoly has used TTA testing?

**Comment by a13383622838** [comment]
I also want to know how 2842656 MAX_STER=85 epochs is calculated. Thank you.

**Comment by HeinzJiao** [comment]
Thank you for your interest. TTA is not used. The max iters and training epochs is determined empirically based on the average precision (AP) observed on the val set (for CrowdAI, the val set is the same as the test set). Specifically, if the AP does not improve for 10 consecutive epochs, the learning rate is decreased by 0.1. If there is still no improvement for another 10 epochs, training is stopped.

I’m currently quite busy, but I will update and improve the repository once this busy period is over.

**Comment by a13383622838** [comment]
> Thank you for your interest. TTA is not used. The max iters and training epochs is determined empirically based on the average precision (AP) observed on the val set (for CrowdAI, the val set is the same as the test set). Specifically, if the AP does not improve for 10 consecutive epochs, the learning rate is decreased by 0.1. If there is still no improvement for another 10 epochs, training is stopped.
> 
> I’m currently quite busy, but I will update and improve the repository once this busy period is over.

Thank you for your reply. I have 10,000 training images of 256×256, and I set the training epochs to 200, but I don't know when overfitting will occur. How can I determine the most suitable training epoch?
